index.html

```diff
@@ -69,8 +69,11 @@ <h3>Water level: <span id="water-level"></span></h3>
   <svg id="visual" preserveAspectRatio="none" viewBox="0 0 1000 1000">
     <rect id="sky" class="sky" />
     <rect id="ground" y="500" />
-    <path id="flow-area" class="sky" />
-    <rect id="water" />
+    <mask id="flow-area-mask">
+      <path id="flow-area" />
+    </mask>
+    <rect id="flow-butt" mask="url(#flow-area-mask)" x="0" y="0" style="width: 100%; height:100%" class="sky" />
```

Comment:
ja gozer, zo gaan we dit niet doen!
```suggestion
    <rect id="flow-square" mask="url(#flow-area-mask)" x="0" y="0" style="width: 100%; height:100%" class="sky" />
```